cle/backends/pe/pe.py

```diff
@@ -62,7 +64,7 @@ def __init__(self, *args, **kwargs):
         self._symbol_cache = self._exports # same thing
         self._handle_imports()
         self._handle_exports()
-        self._handle_relocs()
+        self.__register_relocs()
```

Comment:
Just matching [ELF's](https://github.com/angr/cle/blob/08e9115033da89c0a4691b228def56f68948069d/cle/backends/elf/elf.py#L483). There's a few in there like that. If you'd rather eliminate them, I can update ELF's to go along with this PR.

(For some reason, this comment said it posted... but didn't.)